src/test/java/org/folio/entlinks/controller/delegate/suggestion/LinksSuggestionServiceDelegateHelperTest.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.entlinks.controller.delegate.suggestion;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.folio.entlinks.domain.dto.AuthoritySearchParameter;
+import org.folio.spring.test.type.UnitTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@UnitTest
+@ExtendWith(MockitoExtension.class)
+class LinksSuggestionServiceDelegateHelperTest {
+
+  @Mock
+  private LinksSuggestionsByAuthorityNaturalId linksSuggestionsByAuthorityNaturalId;
+  @Mock
+  private LinksSuggestionsByAuthorityId linksSuggestionsByAuthorityId;
+
+  @InjectMocks
+  private LinksSuggestionServiceDelegateHelper delegateHelper;
+
+  @Test
+  void getDelegate_positive_id() {
+    var actual = delegateHelper.getDelegate(AuthoritySearchParameter.ID);
+    assertThat(actual).isInstanceOf(LinksSuggestionsByAuthorityId.class);
+  }
+
```

Comment:
Please add test for null. We should consider to have some default service.

Reply:
Added test cases.
API has the default value of "NATIRAL_ID" and returns 400 on invalid enum parameter value. Do you think we need to provide default delegate in helper anyway?

Reply:
No. I think it's okay to have it like now.